packages/maskbook/src/extension/options-page/DashboardRouters/Plugins.tsx

```diff
@@ -6,6 +6,7 @@ import PluginCard from '../DashboardComponents/PluginCard'
 import DashboardRouterContainer from './Container'
 import { PluginUI } from '../../../plugins/PluginUI'
 import { PluginScope } from '../../../plugins/types'
+import { useActivatedPluginsSNSAdaptor } from '@masknet/plugin-infra'
```

Comment:
Actually, this only lists the "enabled" plugins, but since we don't have that functionality yet, I will handle this in #3066